<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>791</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_file">
         <property name="text">
          <string>Адрес файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entry_file">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>30</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Введите путь до файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_button">
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_button">
         <property name="text">
          <string>Удалить объект</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
